src/vcpkg/binarycaching.cpp

```diff
@@ -2500,10 +2510,12 @@ namespace vcpkg
 
             if (!s.upkg_templates_to_get.empty())
             {
+                ZipTool zip_tool;
```

Comment:
This should move into the block starting on 2423 to use the zip_tool from there rather than repeating its setup.

Reply:
This comment was marked resolved but it's still not fixed.